tools/nvqpp/nvq++.in

```diff
@@ -112,7 +112,10 @@ function show_help {
 	Enable the verbose option (-v) to the C++ compiler.
 
 --load=<domain>
-    Load the domain-specific library, e.g. chemistry.
+	Load the domain-specific library, e.g. chemistry.
+
+--mapping_file <path/to/file>
```

Comment:
I actually do think that we should retain the dash for command line arguments.

Reply:
Having different strings for Python vs command line seems undesirable to me (from a maintenance and user perspective), but I can make that change if I am understanding that correctly. Let me post an update for the sake of specificity and you can let me know what you think.

Reply:
Updated in 5c9446c564c227d5082f61455dedce6c9ebe79ec. Note that I still retained `mapping_file` in the string blob that goes in the executable (`NVQPP_TARGET_BACKEND_CONFIG`) so that the runtime can be looking for the same key.